docs/tutorials/ui-customization.md

```diff
@@ -61,7 +61,7 @@ The following elements can be added to the UI bar using this configuration value
   starts playing the presentation at an increased speed
 * spacer: adds a chunk of empty space between the adjacent elements.
 * picture_in_picture: adds a button that enables/disables picture-in-picture mode on browsers
-  that support it. Button is invisible on other browsers.
+  that support it. Button is invisible on other browsers. Note that it will use the [Document Picture-in-Picture API]() if supported.
```

Comment:
This tutorial actually lists the description of `picture_in_picture` separately in three different places. So it's kind of a pain in the ass, but all three have to be updated.

Reply:
Oops! Thanks! Fixed in https://github.com/shaka-project/shaka-player/pull/4969/commits/6b68bb34dfe127a4e2cb22f058dcb9de3545fe92